.github/workflows/test.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - run: npm test -- -l
       - name: Upload tests report(cypress mochaawesome merged HTML report)
         if: ${{ always() }}
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
         with:
           name: report
           path: reports
```

Comment:
Ensure that the `reports` directory exists and contains the test reports you intend to upload. Otherwise, this step might fail.